apps/api/env/.env.testnet

```
@@ -0,0 +1,2 @@
+RPC_NODE_ENDPOINT=https://testnetrpc.akashnet.net:443
+DEPLOYMENT_GRANT_DENOM=uakt
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)